src/redfish_collector/routers/prometheus.py

```
@@ -2,107 +2,98 @@
-                        server['password'] = str(password)
-                        server['timeCalled'] = time.time()
-                        existed = True
-                    # if float(server['timeCalled']) > time.time() - 300:
-                    #     inventory.remove(server)
-                if existed is False:
-                    inventory.append(block)
-                with open('%sinventory.yml' %REDFISH_DATA, 'w') as f:
-                    yaml.dump(inventory, f, default_flow_style=False)
+                with open(inventoryFile, 'w') as f:
+                    yaml.dump([block], f, default_flow_style=False)
         except Exception as err:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)